homeassistant/components/wallbox/__init__.py

```diff
@@ -125,11 +127,10 @@ def _get_data(self) -> dict[str, Any]:
             data[CHARGER_STATUS_DESCRIPTION_KEY] = CHARGER_STATUS.get(
                 data[CHARGER_STATUS_ID_KEY], ChargerStatus.UNKNOWN
             )
-
             return data
 
         except requests.exceptions.HTTPError as wallbox_connection_error:
```

Comment:
We need to catch `ConnectionError` here since that's raised by `self._authenticate`.

Reply:
@MartinHjelmare I changed this -> added an additional catch.